### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by Stevendeo**

### CODE REVIEW

File: .github/workflows/build_docker.yml [hunk 1]
```diff
@@ -26,6 +30,9 @@ jobs:
       # to `ocaml` with the container parameters
       - run: sudo chmod a+wx .
 
+      - name: Reclaim cache directory
+        run: sudo chown -R 1001:1001 .
```
Comment (Gbury) [inline .github/workflows/build_docker.yml]: what kind of problem does that fix ?

**Review by Gbury — COMMENTED** [review]
what's the exact motivation for this PR ?

**Comment by Stevendeo** [comment]
The CI is currently broken, this PR aims at fixing the different CI issues.

File: .github/workflows/build_docker.yml (hunk 1 above)
Comment (Stevendeo) [reply .github/workflows/build_docker.yml]: The docker image build fails ; right now the error is `Error: EACCES: permission denied, mkdir '/__w/_temp/9c8f979e-f394-49cf-ae44-be4cb680bb8a'` so I guessed there was some kind of ownership issue. Right now, I'm playing with CI configuration to start the Docker check on a specific set of PRs (the PR of branches named `fix-ci`).

**Comment by hra687261** [comment]
This PR needs to be merged otherwise all new PRs and PRs that have recent commits will keep having failing CIs

**Review by Gbury — APPROVED** [review]

**Comment by Gbury** [comment]
Well, looks like this actually broke CI once merged cc @Stevendeo , :p

**Comment by hra687261** [comment]
It actually failed in a **new** way. The error is not the same as the one it fixes.
I created a PR #525 which should fix the new CI error

**Pull request merged**